bluebutton-data-pipeline-fhir-load/src/main/java/gov/hhs/cms/bluebutton/datapipeline/fhir/transform/DataTransformer.java

```diff
@@ -951,7 +974,60 @@ private TransformedBundle transformBeneficiary(RifRecordEvent<BeneficiaryRow> ri
 		beneficiary.addIdentifier().setSystem(CODING_SYSTEM_CCW_BENE_ID).setValue(record.beneficiaryId);
 		beneficiary.addAddress().setState(record.stateCode).setDistrict(record.countyCode)
 				.setPostalCode(record.postalCode);
-		// TODO rest of mapping
+		if (record.birthDate != null) {
+			beneficiary.setBirthDate(Date.valueOf(record.birthDate));
+		}
+
+		switch (record.sex) {
+		case ('M'):
+			/* Male */
+			beneficiary.setGender((AdministrativeGender.MALE));
+			break;
+		case ('F'):
+			/* Female */
+			beneficiary.setGender((AdministrativeGender.FEMALE));
+			break;
+		default:
+			/* Unknown */
+			beneficiary.setGender((AdministrativeGender.UNKNOWN));
+			break;
+
+		}
+		// TODO Could not map the following fields. Initially created extension
+		// for these fields but decided this may not be the best approach.
+		// Have created a JIRA ticket called "Finalize fields for Beneficary" to
+		// revisit on where to best map the following fields.
+		/*
+		 * beneficiary.addExtension().setUrl(CODING_SYSTEM_CCW_BENE_RACE_CD)
+		 * .setValue(new
+		 * Coding().setSystem(CODING_SYSTEM_CCW_BENE_RACE_CD).setCode(String.
+		 * valueOf(record.race)));
+		 * 
+		 * beneficiary.addExtension().setUrl(
+		 * CODING_SYSTEM_CCW_BENE_ENTLMT_RSN_ORIG) .setValue(new
+		 * Coding().setSystem(CODING_SYSTEM_CCW_BENE_ENTLMT_RSN_ORIG)
+		 * .setCode(String.valueOf(record.entitlementCodeOriginal)));
+		 * 
+		 * beneficiary.addExtension().setUrl(
+		 * CODING_SYSTEM_CCW_BENE_ENTLMT_RSN_CURR) .setValue(new
+		 * Coding().setSystem(CODING_SYSTEM_CCW_BENE_ENTLMT_RSN_CURR)
+		 * .setCode(String.valueOf(record.entitlementCodeCurrent)));
+		 * 
+		 * beneficiary.addExtension().setUrl(CODING_SYSTEM_CCW_BENE_ESRD_IND)
+		 * .setValue(new Coding().setSystem(CODING_SYSTEM_CCW_BENE_ESRD_IND)
+		 * .setCode(String.valueOf(String.valueOf(record.
+		 * endStageRenalDiseaseCode))));
+		 * 
+		 * beneficiary.addExtension().setUrl(
+		 * CODING_SYSTEM_CCW_BENE_MDCR_STATUS_CD) .setValue(new
+		 * Coding().setSystem(CODING_SYSTEM_CCW_BENE_MDCR_STATUS_CD)
+		 * .setCode(String.valueOf(record.medicareEnrollmentStatusCode)));
+		 */
+
+		beneficiary.addName().addGiven(record.nameGiven).setUse(HumanName.NameUse.USUAL);
+		beneficiary.addName().addGiven((String.valueOf(record.nameMiddleInitial))).setUse(HumanName.NameUse.USUAL);
+		beneficiary.addName().addFamily(record.nameSurname).setUse(HumanName.NameUse.USUAL);
```

Comment:
These three lines are bugged: you're adding three separate Name elements, each with one field filled in.